plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -65,6 +65,22 @@ func (c *CachedNodeInfo) GetNodeInfo(id string) (*api.Node, error) {
 	return node.(*api.Node), nil
 }
 
+// podMetadata defines a type, that is an expected type that is passed
```

Comment:
you could just say "podMetadata is a type that is passed as metadata for predicate functions"